apps/institution-service/cypress/support/e2e.ts

```diff
@@ -16,43 +17,65 @@
 // Import commands.js using ES2015 syntax:
 import { JwtPayload } from "jsonwebtoken";
 import "./commands";
+import {
+  DefaultPermissions,
+  UiClientPermissions,
+} from "../../src/shared/enums";
 
 before(() => {
+  const widgetAudience = Cypress.env("AUTH0_WIDGET_AUDIENCE") as string;
+  const clientAudience = "ucp-hosted-apps";
+
+  const username = Cypress.env("E2E_INSTITUTION_USERNAME") as string;
+  const password = Cypress.env("E2E_INSTITUTION_PASSWORD") as string;
+
+  const ucpWebUiClientId = Cypress.env("WEB_UI_CLIENT_ID") as string;
+
   const client_id = Cypress.env("WIDGET_CLIENT_ID") as string;
   const client_secret = Cypress.env("WIDGET_CLIENT_SECRET") as string;
-  const audience = Cypress.env("AUTH0_WIDGET_AUDIENCE") as string;
+
+  const scope = `${Object.values(DefaultPermissions).join(" ")} ${Object.values(UiClientPermissions).join(" ")}`;
+
+  cy.request({
+    method: "POST",
+    url: `https://${Cypress.env("AUTH0_DOMAIN")}/oauth/token`,
+    body: {
+      audience: clientAudience,
+      client_id: ucpWebUiClientId,
+      grant_type: "password",
+      password,
+      username,
+      scope,
+    },
+  }).then((response: Cypress.Response<JwtPayload>) => {
+    Cypress.env("USER_ACCESS_TOKEN", response.body.access_token);
+  });
 
   cy.request({
     method: "POST",
     url: `https://${Cypress.env("AUTH0_DOMAIN")}/oauth/token`,
     body: {
       grant_type: "client_credentials",
-      audience,
+      audience: widgetAudience,
       client_id,
       client_secret,
     },
   }).then((response: Cypress.Response<JwtPayload>) => {
     Cypress.env("ACCESS_TOKEN", response.body.access_token);
   });
 
-  const username = Cypress.env("E2E_INSTITUTION_USERNAME") as string;
-  const password = Cypress.env("E2E_INSTITUTION_PASSWORD") as string;
-  const ucpWebUiClientId = Cypress.env("WEB_UI_CLIENT_ID") as string;
-  const ucpWebUiClientSecret = Cypress.env("WEB_UI_CLIENT_SECRET") as string;
-
   cy.request({
     method: "POST",
     url: `https://${Cypress.env("AUTH0_DOMAIN")}/oauth/token`,
     body: {
       grant_type: "password",
       scope: "openid offline_access profile email",
+      audience: widgetAudience,
+      client_id: ucpWebUiClientId,
       username,
       password,
-      audience: audience,
-      client_id: ucpWebUiClientId,
-      client_secret: ucpWebUiClientSecret,
     },
   }).then((response: Cypress.Response<JwtPayload>) => {
-    Cypress.env("USER_ACCESS_TOKEN", response.body.access_token);
+    Cypress.env("NO_KEYS_USER_ACCESS_TOKEN", response.body.access_token);
```

Comment:
The NO_KEYS name is confusing. It makes it seem like this is a user that doesn't have api key access. I believe what this test is supposed to be testing is that if a user requests with all the right information to get the cacheList that they can't get it. We should request with all the scope and audience necessary to try and get access when we make this request.

Reply:
This code is basically untouched from what existed before this PR. I just renamed some things that I thought were too vague. 

The token that comes from this support function shouldn't have the rights to hit the cacheList endpoint. The test that uses this token is testing that a 403 should be returned.

I'll rename this token, since it actually represents a user who does not have access to interact with the widget api (in the auth0 context)